components/automate-chef-io/content/docs/eas.md

```diff
@@ -0,0 +1,108 @@
++++
+title = "Chef EAS"
+description = "The Chef Software Enterprise Applications Stack"
+date = 2019-10-18T18:54:09+00:00
+draft = true
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "applications"
+    weight = 10
++++
+
+This page combines 'concept' and 'task' documentation.
+Ideally, we should have two pages
+Concept structure:
+Concept
+    sections
+        examples
+
+Task structure:
+Task
+    prerequisite
+    context
+    section
+    steps
+        - cmd: describes action
+    result (optional, only one, precedes example): Describes expected outcome
+    example(optional, precedes post-req): Illustrates or supports task
+    postreq: What to do next, he next task or related resources.
+
+## The Chef Enterprise Application (EAS) Stack
+
+The Chef Enterprise Application Stack (EAS) provides the tools for defining and managing your application throughout its entire lifecycle.
+Chef EAS application, infrastructure, and security automation helps your organization manage the lifecycle of your application with stable, consistent, and repeatable processes, letting you focus on delivering new features--and new value--to your customers.
+Chef EAS automates your application's build and release stages, the configuration for its underlying environments and infrastructure, audits both your application and its infrastructure for security and compliance, while also providing a uniform user experience and visibility accross your entire system.
+
+1. Chef Habitat automates how you build and deploy your applications because delivering applications is where you derive your business value
```

Comment:
so it kinda jumped out of me that most of these say _automates_ and then automate doesn't .. and I know thats a product name issue.. but since this in the automate docs I wonder if a different word may be better here

Reply:
Agree. I think I'm going to use the word marks & drop the ordered list, too.